_posts/2017-03-08-this-week-in-rustfest-8-tickets-on-sale.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "This Week in RustFest -8: Tickets on Sale, more information on Kyiv"
+socialImageSrc: '/assets/posts/2017-tickets-full.png'
+authors:
+  - ben
+  - andrey
+---
+
+It's just eight more weeks to go for RustFest 2017 in Kyiv. Time to get started with our regular compilations of updates in any given week. And what a week this was.
+
+### Buy your Regular Ticktes now
+
+First and foremost, we've opened ticket sales. Since last friday anyone can get their very own admission ticket to the conference for a limited time offer of 89EUR.
+
+[![Tickets are on sale](/assets/posts/2017-tickets-full.png)](http://2017.rustfest.eu/)
+
+### CfP and Keynotes
+
+On Sunday we've closed our CfP and have started the proposals. Once again we got many more proposals than we can accommodate, but it will be tougher than ever as the submissions are very impressive and diverse. We've left the question of length per talk a bit more open this time, allowing the authors to suggest whether they'll need 15 or 30minutes and we are tinkering with the idea of presenting a lightning-talk like section of those 15min talks. More on all that once we've decided on the final schedule.
```

Comment:
* "have started the proposal review"
* "30 minutes" (space between 30 and minutes, same for 15min)